lib/profiler.js

```diff
@@ -97,19 +99,21 @@ function writeProfileFiles(profilePath, options) {
       /* after %~~%profile%~~% before %~~%versions%~~% */ htmlParts[2] +
       versions +
       /* after %~~%versions%~~% */ htmlParts[4];
-    writeFileSync(profilePath + ".html", htmlResult);
+    const htmlPath = profilePath + ".html";
+    writeFileSync(htmlPath, htmlResult);
+    open && opn(htmlPath);
```

Comment:
Can you please add a comment that this will open the browers?

e.g.

```suggestion
    if (options.open) {
       // Open default browser
       opn(htmlPath);
    }
```